addons/binding/org.openhab.binding.solarlog/src/main/java/org/openhab/binding/solarlog/internal/HttpUtils.java

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
It seems that this file has a missing  or invalid copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.